tests/cephadm/test_label_no_schedule.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+
+
+class LabelNoScheduleError(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """verify "__no_schedule" label
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    installer_node = ceph_cluster.get_nodes(role="installer")
+    node = ceph_cluster.get_nodes()[1]
+    daemon_type, node_name, label = "osd", node.hostname, "_no_schedule"
+    exp_out = f"Added label _no_schedule to host {node_name}"
+    # Adding "__no_schedule" label and checking daemons to node.
+    result = CephAdm(installer_node).ceph.orch.label.add(node_name, label)
```

Comment:
can we enhance verification with another check to verify the `_no_schedule` present in `ceph orch host ls <hostname>` .

Reply:
Thanks @sunilkumarn417 

Updated

https://github.com/red-hat-storage/cephci/pull/2009/files#diff-0d18e68d5971d342ef2e234b4133953118d1d8372709bc5a1388ebda492a3016

https://github.com/red-hat-storage/cephci/pull/1990/files#diff-5dfa94d059d71b3d5a0ffc1be352b868511fea632cedf09fa8da33b71442456aR30